spec/requests/v1/sessions_spec.rb

```diff
@@ -113,19 +119,25 @@
         }
       end
 
-      it "refreshs the jwt token" do
+      it "refreshes the jwt token" do
         sleep(1)
-        expect { post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: { "Cookie" => response['Set-Cookie'] } }.to change { response.cookies.with_indifferent_access[:jwt] }
-        expect(jwt_cookie).to be_truthy
+        expect { post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: auth_header }.to change { response.cookies.with_indifferent_access[:jwt] }
+      end
+
+      it "refreshes the refresh token" do
+        sleep(1)
+        expect { post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: auth_header }.to change { response.cookies.with_indifferent_access[:refresh] }
       end
 
       it 'returns jwt token and json response' do
-        post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: { "Cookie" => response['Set-Cookie'] }
+        sleep(1)
+        post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: { "Authorization" => "Bearer #{response.cookies.with_indifferent_access[:refresh]}" }
         expect(response.status).to eq(200)
         expect(json_response['email']).to eq(user.email)
         expect(json_response['participants']).to eq []
         expect(json_response['type']).to eq []
         expect(jwt_cookie).to be_truthy
+        expect(response.cookies.with_indifferent_access[:refresh]).to be_truthy
```

Comment:
`refresh_token` is available so this could be cleaned up:
```suggestion
        expect(refresh_token).to be_truthy
```